vtex-apps/popover-layout
# Add backdrop and onScroll props to popover-layout

#### What does this PR do? \*

As the title says

#### How to test it? \*

1. Go to the [workspace](https://popoverswitcher--storecomponents.myvtex.com/)
2. Below the shelf, you will see three `locale-swticher`s
3. Click in every one of them

Here is the configuration of every `popover-layout`
1.
```jsonc
{
  "popover-layout#locale-switcher": {
    "children": [
      "locale-switcher-list"
    ],
    "props": {
      "onScroll": "close",
      "backdrop": "clickable"
    }
  }
}
``` 

2.
```jsonc
{
  "popover-layout#locale-switcher2": {
    "children": [
      "locale-switcher-list"
    ],
    "props": {
      "onScroll": "block",
      "backdrop": "clickable"
    }
  }
}
``` 

3.
```jsonc
{
  "popover-layout#locale-switcher3": {
    "children": [
      "locale-switcher-list"
    ]
  }
  // the default props are onScroll = 'none' and backdrop = 'none'
}
``` 

Note that the `popover-layout` is in the middle of the screen, that's because the trigger has the whole screen size, so the `popover-layout` will be rendered at the middle of it

#### Related to / Depends on \*
